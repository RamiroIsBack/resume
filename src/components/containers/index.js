import IntroContainer from "./IntroContainer.js";

import SummaryContainer from "./SummaryContainer.js";

import ServicesContainer from "./ServicesContainer.js";

import TimeLineContainer from "./TimeLineContainer.js";

import SideBarContainer from "./SideBarContainer.js";

import NombreContainer from "./NombreContainer";

import FooterContainer from "./FooterContainer.js";

import MapaContainer from "./MapaContainer.js";

import JobsListContainer from "./JobsListContainer";

import ModalWorkContainer from "./ModalWorkContainer";

export {
  ModalWorkContainer,
  JobsListContainer,
  MapaContainer,
  FooterContainer,
  NombreContainer,
  SummaryContainer,
  IntroContainer,
  ServicesContainer,
  TimeLineContainer,
  SideBarContainer
};
