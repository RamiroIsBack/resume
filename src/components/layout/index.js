import App from "./App.js";
import JobsLayout from "./JobsLayout";
export { App, JobsLayout };
