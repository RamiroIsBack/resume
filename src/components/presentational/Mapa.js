import React, { Component } from 'react'
import { withScriptjs, withGoogleMap, GoogleMap, Marker,  InfoWindow, } from 'react-google-maps'

class Mapa extends Component {
  constructor(){
    super();
    this.state={
      mapa: null
    };
  }

  makeMarker(){
    let foto ='';
    let marker = {};
    if(this.props.mapInfo){
      foto = this.props.mapInfo.urlPic;
    }
    marker =(
      <Marker
        position={{
          lat: 39.3311347
          ,
          lng: -76.6450283
        }}
        onClick={() => this.handleClick('https://www.google.com/maps/place/Hampden,+Baltimore,+Maryland+21211/@39.3399061,-76.6020314,12.93z/data=!4m5!3m4!1s0x89c804d5f7905ff3:0x5cc1e5ec25566796!8m2!3d39.3333706!4d-76.632607')}
      >

        <InfoWindow >
          <div style = {{maxWidth: 600,padding : 0}}>

            <img role='presentation' src={foto} className ='infoWindow__img' alt='foto' id='fotoInfoWindow'
              style = {{
                opacity: 0.9, maxHeight: 60, maxWidth:60, cursor: 'pointer',float:'left', marginRight:'2px',
              }}
              onClick={() => this.handleClick('https://www.google.com/maps/place/Hampden,+Baltimore,+Maryland+21211/@39.3399061,-76.6020314,12.93z/data=!4m5!3m4!1s0x89c804d5f7905ff3:0x5cc1e5ec25566796!8m2!3d39.3333706!4d-76.632607')}>
            </img>

            <div className='infoWindow__bio' style = {{padding : 0,float:'right'}} >
              <h6 style = {{marginTop : 2,marginBottom:2}}>I'm in</h6>
              <h5 style = {{marginTop : 2,marginBottom:0}}>Hampden</h5>
              <h6 style = {{marginTop : 2,marginBottom:2}}>Baltimore, MD</h6>
            </div>

          </div>
        </InfoWindow>

      </Marker>

    );




    return marker;
  }
  handleClick(urlToGo){
    window.open(urlToGo,'_blank');
  }


  render() {

    let marker = this.makeMarker();
    return (

      <GoogleMap

        defaultZoom={9}
        defaultCenter={{  lat: 39.4832391, lng: -76.660066  }}
      >
        {marker}
      </GoogleMap>
    )
  }
}

export default withGoogleMap(Mapa)
