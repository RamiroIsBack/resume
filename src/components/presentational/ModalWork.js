import React from "react";
import style from "./styles";

class ModalWork extends React.Component {
  constructor() {
    super();
    this.state = {
      recibir: true
    };
  }

  gestionaIrACreacion(event) {
    if (event.target.id) {
      this.props.onSelectCreaciones(event.target.id);
    } else {
      this.props.onSelectCreaciones("allCreaciones");
    }
  }
  gestionaIrAFeria(event) {
    this.props.onSelectFerias("allFerias");
  }
  render() {
    // Render nothing if the 'show' prop is false
    if (!this.props.show) {
      return null;
    }

    var tipoList = this.props.creacionList.map((tipo, i) => {
      let estilo = {
        cursor: "pointer",
        color: "black",
        backgroundColor: "transparent",
        textDecoration: "none",
        padding: 0
      };

      return (
        <div key={i} style={{ padding: 0 }}>
          <div className="col-xs-offset-3 col-xs-9" style={{ padding: 1 }}>
            <a
              id={tipo.nombre}
              onClick={this.gestionaIrACreacion.bind(this)}
              to="/Diseños"
              style={estilo}
            >
              <h5 id={tipo.nombre}>{tipo.nombre}</h5>
            </a>
          </div>
        </div>
      );
    });

    var stiloModal = {
      position: "absolute",
      backgroundColor: "white",
      maxWidth: "100%",
      minHeight: 300,
      maxHeight: "100%",
      margin: "0 auto",
      //To make sure that it is really on top of all other elements, we set z-index to a high value (by default all elements have a z-index value of 0).
      zIndex: 6668,
      top: "20px",
      left: "2px",
      right: "70px",
      bottom: "40px",
      overflow: "auto",
      WebkitOverflowScrolling: "touch",
      borderRadius: "5px",
      outline: "none"
    };

    return (
      <div style={style.modal.backdropStyle}>
        <div style={stiloModal}>
          <div>
            <div
              className="col-xs-12 col-sm-8 col-md-8 col-lg-8"
              style={style.modal.formContainer}
            >
              <button
                onClick={this.props.onClose}
                className="btn glyphicon glyphicon-remove pull-right"
                style={style.modal.btnClose}
              />
              <div className="col-xs-offset-1 col-xs-8">
                <a
                  id="allCreaciones"
                  name="allCreaciones"
                  onClick={this.gestionaIrACreacion.bind(this)}
                  to="/Diseños"
                  style={{
                    cursor: "pointer",
                    color: "black",
                    backgroundColor: "transparent",
                    textDecoration: "none",
                    padding: 0
                  }}
                >
                  <h4 id="allCreaciones" name="allCreaciones">
                    {" "}
                    TIENDA:
                  </h4>
                </a>
              </div>

              <div>{tipoList}</div>

              <div className="col-xs-offset-1 col-xs-8">
                <a
                  to="/Conocenos"
                  style={{
                    cursor: "pointer",
                    color: "black",
                    backgroundColor: "transparent",
                    textDecoration: "none",
                    padding: 0
                  }}
                  onClick={this.props.onClose}
                >
                  <h4> SOBRE MI</h4>
                </a>
              </div>

              <div className="col-xs-offset-1 col-xs-8">
                <a
                  to="/Taller"
                  style={{
                    cursor: "pointer",
                    color: "black",
                    backgroundColor: "transparent",
                    textDecoration: "none",
                    padding: 0
                  }}
                  onClick={this.props.onClose}
                >
                  <h4> TALLER</h4>
                </a>
              </div>

              <div className="col-xs-offset-1 col-xs-8">
                <a
                  id="creaciones"
                  to="/Ferias"
                  style={{
                    cursor: "pointer",
                    color: "black",
                    backgroundColor: "transparent",
                    textDecoration: "none",
                    padding: 0
                  }}
                  onClick={this.props.onClose}
                >
                  <h4> FERIAS</h4>
                </a>
              </div>

              <div className="col-xs-offset-1 col-xs-8">
                <a
                  to="/Contacto"
                  style={{
                    cursor: "pointer",
                    color: "black",
                    backgroundColor: "transparent",
                    textDecoration: "none",
                    padding: 0
                  }}
                  onClick={this.props.onClose}
                >
                  <h4> CONTACTO</h4>
                </a>
              </div>
            </div>
          </div>
        </div>
      </div>
    );
  }
}

//I tihk I don't need this cos it's only to put children to it
/*Modal.propTypes = {
  onClose: PropTypes.func.isRequired,
  show: PropTypes.bool,
  children: PropTypes.node
}*/

export default ModalWork;
