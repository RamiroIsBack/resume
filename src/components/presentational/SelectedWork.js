import React, { Component } from "react";
import Works_css from "../../css"; // eslint-disable-line no-unused-vars

class SelectedWork extends Component {
  render() {
    return (
      <div>
        <h2>{this.props.copy.nombre}</h2>
        <p>{this.props.copy.bio}</p>
      </div>
    );
  }
}

export default SelectedWork;
