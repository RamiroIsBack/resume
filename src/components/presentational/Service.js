import React, {Component} from 'react';
import LazyLoad from 'react-lazy-load';

class Service extends Component {

  handleClick(){
    if(this.props.serviceInfo.urlLink){
      window.open(this.props.serviceInfo.urlLink,'_blank');
    }
  }

  render() {
    let logo= '';
    let cursor = {};
    let stylingBarWidth =  { width:'80%'};
    if(this.props.serviceInfo){
      logo = this.props.serviceInfo.urlPic;
      let percentage = parseInt(this.props.serviceInfo.percentage,10);
      if (this.props.sectionSelected){
        stylingBarWidth ={
          width: ''+percentage+'%',
          animationName: 'progress',
          animationDuration: '3s',
          animationTimingFunction: 'ease-in-out',
        };
      }else {
        stylingBarWidth ={
          width: ''+percentage+'%',

        };
      }
      if(this.props.serviceInfo.urlLink){
        cursor={
          cursor:'pointer',
        };
      }
    }

    return (
      <div >
        <div className = 'name__pic__container'>
          <h2  className = 'service__name' >{this.props.serviceInfo.nombre}</h2>
          <LazyLoad height={60} offsetVertical={100}>
            <img className = 'logo_pic'
              style= {cursor}
              alt=''
              src= {logo}
              onClick = {this.handleClick.bind(this)}
            ></img>
          </LazyLoad>
        </div>
        <div className = 'progress__bar__container'>
          <div className = 'progress__bar' style={stylingBarWidth}></div>
        </div>
        <h5 className = 'percentage__num'>{this.props.serviceInfo.percentage}%</h5>
        <div className = 'bio__container__services'>
          {this.props.serviceInfo.bio.split('\n').map((item, key) => {
            return <span key={key}>{item}<br/></span>;})}
        </div>
      </div>
    );
  }
}

export default Service;
