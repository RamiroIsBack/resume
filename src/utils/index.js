import Firebase from './Firebase.js';
import PopularFunctions from './PopularFunctions.js';
export {
  Firebase,
  PopularFunctions,
};