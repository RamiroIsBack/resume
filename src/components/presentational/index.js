import Service from "./Service.js";
import TimeBox from "./TimeBox.js";
import Mapa from "./Mapa.js";
import Work from "./Work.js";
import SelectedWork from "./SelectedWork.js";
import ModalWork from "./ModalWork.js";
export { Mapa, Service, TimeBox, Work, SelectedWork, ModalWork };
