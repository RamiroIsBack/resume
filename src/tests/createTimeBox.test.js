/**
 * @jest-environment node
 */

import React from 'react';
import renderer from 'react-test-renderer';
import {TimeBox} from '../components/presentational'
import {Firebase} from '../utils';

it('getTimeBoxComponents', () => {
  return Firebase.getCopyForTesting()
    .then(result => { //the return is a promise: result(copyList)
      let component = {};
      let timeBoxComponentList = [];
      let rawList = result;
      let timeBoxRawListObject = {};
      for(let i = 0; i<rawList.length; i++){
        if(rawList[i].nombre ==='timeLine'){
          timeBoxRawListObject = rawList[i];
        }
      }
      for (var property in timeBoxRawListObject) {
        if (timeBoxRawListObject.hasOwnProperty(property)) {
          if (property!=='nombre' && property !=='id'){
            //console.log(JSON.stringify(property));
            component = renderer.create(
              <div className = {timeBoxRawListObject[property].className} key ={property}>
                <TimeBox
                  urlClick ='handleClickMethod'
                  copy = {timeBoxRawListObject[property]}
                  screenSize = {700}
                />
              </div>
            ).toJSON();
          }
        }
        timeBoxComponentList.push( component   );
      }

      expect (timeBoxComponentList).toMatchSnapshot();

    });
});
// @magnusart got it right
// https://www.bountysource.com/issues/41190983-syntax-error-invalid-or-unexpected-token-with-png
