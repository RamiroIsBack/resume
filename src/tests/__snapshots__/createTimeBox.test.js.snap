// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getTimeBoxComponents 1`] = `
Array [
  Object {},
  <div
    className="studies__android"
>
    <div
        className="leaf__container"
    >
        <div
            className="timeBox__headline__container"
        >
            <h3
                className="timeBox__name"
                onClick={[Function]}
                style={
                    Object {
                        "cursor": "pointer",
                      }
                }
            >
                Android development learning
            </h3>
            <div
                className="logo__photo__container"
            >
                <img
                    className="logo__photo"
                    onClick={[Function]}
                    src="https://firebasestorage.googleapis.com/v0/b/resume-40a8a.appspot.com/o/fotos%2Fandroid.png?alt=media&token=dbd89d37-fd48-4257-aab4-53122db4ac39"
                    style={
                        Object {
                            "cursor": "pointer",
                          }
                    }
                />
            </div>
        </div>
        <div
            className="fecha_container"
        >
            <h3
                className="fecha__inicio"
            >
                2013
                 -
            </h3>
            <h3
                className="fecha__fin"
            >
                -&gt;  
                2014
            </h3>
        </div>
        <div
            className="description__container"
        >
            <span
                className="description__text"
            >
                I got to be a self taught android developer while I run my, so far, first and only start-up.
                <br />
            </span>
        </div>
    </div>
</div>,
  <div
    className="studies__civil__servant"
>
    <div
        className="leaf__container"
    >
        <div
            className="timeBox__headline__container"
        >
            <h3
                className="timeBox__name"
                onClick={[Function]}
                style={
                    Object {
                        "cursor": "auto",
                      }
                }
            >
                Civil servant studies
            </h3>
            <div
                className="logo__photo__container"
            >
                <img
                    className="logo__photo"
                    onClick={[Function]}
                    src="https://firebasestorage.googleapis.com/v0/b/resume-40a8a.appspot.com/o/fotos%2Fcivil.jpg?alt=media&token=ed72cab4-6e35-43b5-a7cf-2f3ff1a2cf7f"
                    style={
                        Object {
                            "cursor": "auto",
                          }
                    }
                />
            </div>
        </div>
        <div
            className="fecha_container"
        >
            <h3
                className="fecha__inicio"
            >
                2008
                 -
            </h3>
            <h3
                className="fecha__fin"
            >
                -&gt;  
                2010
            </h3>
        </div>
        <div
            className="description__container"
        >
            <span
                className="description__text"
            >
                for some time I got prepared to become a civil servant in Madrid, Spain.
                <br />
            </span>
        </div>
    </div>
</div>,
  <div
    className="studies__degree"
>
    <div
        className="leaf__container"
    >
        <div
            className="timeBox__headline__container"
        >
            <h3
                className="timeBox__name"
                onClick={[Function]}
                style={
                    Object {
                        "cursor": "pointer",
                      }
                }
            >
                Computer science degree
            </h3>
            <div
                className="logo__photo__container"
            >
                <img
                    className="logo__photo"
                    onClick={[Function]}
                    src="https://firebasestorage.googleapis.com/v0/b/resume-40a8a.appspot.com/o/fotos%2Fuax.png?alt=media&token=d7927956-96fe-4a08-b399-32038af344a2"
                    style={
                        Object {
                            "cursor": "pointer",
                          }
                    }
                />
            </div>
        </div>
        <div
            className="fecha_container"
        >
            <h3
                className="fecha__inicio"
            >
                2000
                 -
            </h3>
            <h3
                className="fecha__fin"
            >
                -&gt;  
                2005
            </h3>
        </div>
        <div
            className="description__container"
        >
            <span
                className="description__text"
            >
                Bachelor degree in computer science at Univerdad Alfonso DÃ©cimo el sabio, Madrid, Spain.
                <br />
            </span>
        </div>
    </div>
</div>,
  <div
    className="studies__vertical"
>
    <div
        className="leaf__container"
    >
        <div
            className="timeBox__headline__container"
        >
            <h3
                className="timeBox__name"
                onClick={[Function]}
                style={
                    Object {
                        "cursor": "pointer",
                      }
                }
            >
                Rope access technician 
            </h3>
            <div
                className="logo__photo__container"
            >
                <img
                    className="logo__photo"
                    onClick={[Function]}
                    src="https://firebasestorage.googleapis.com/v0/b/resume-40a8a.appspot.com/o/fotos%2Fgwo.jpg?alt=media&token=97f84471-61c0-42fc-8925-f97f0958222f"
                    style={
                        Object {
                            "cursor": "pointer",
                          }
                    }
                />
            </div>
        </div>
        <div
            className="fecha_container"
        >
            <h3
                className="fecha__inicio"
            >
                2014
                 -
            </h3>
            <h3
                className="fecha__fin"
            >
                -&gt;  
                2016
            </h3>
        </div>
        <div
            className="description__container"
        >
            <span
                className="description__text"
            >
                Wind turbine generator technician by GWO.
                <br />
            </span>
            <span
                className="description__text"
            >
                IRATA level 2.
                <br />
            </span>
            <span
                className="description__text"
            >
                Both international certifications.
                <br />
            </span>
        </div>
    </div>
</div>,
  <div
    className="studies__react"
>
    <div
        className="leaf__container"
    >
        <div
            className="timeBox__headline__container"
        >
            <h3
                className="timeBox__name"
                onClick={[Function]}
                style={
                    Object {
                        "cursor": "pointer",
                      }
                }
            >
                Web-app development self teaching to full-stack 
            </h3>
            <div
                className="logo__photo__container"
            >
                <img
                    className="logo__photo"
                    onClick={[Function]}
                    src="https://firebasestorage.googleapis.com/v0/b/resume-40a8a.appspot.com/o/fotos%2Fudemy.png?alt=media&token=b10d8ae5-5cf2-4b28-be17-75bd9b960f71"
                    style={
                        Object {
                            "cursor": "pointer",
                          }
                    }
                />
            </div>
        </div>
        <div
            className="fecha_container"
        >
            <h3
                className="fecha__inicio"
            >
                2016
                 -
            </h3>
            <h3
                className="fecha__fin"
            >
                -&gt;  
                never ending
            </h3>
        </div>
        <div
            className="description__container"
        >
            <span
                className="description__text"
            >
                One can never stop learning when it comes to programming and designing ,that's what makes it fun, right?
                <br />
            </span>
            <span
                className="description__text"
            >
                Link goes to one of the many online curses I've done so far.
                <br />
            </span>
        </div>
    </div>
</div>,
  <div
    className="work__english__teacher"
>
    <div
        className="leaf__container"
    >
        <div
            className="timeBox__headline__container"
        >
            <h3
                className="timeBox__name"
                onClick={[Function]}
                style={
                    Object {
                        "cursor": "pointer",
                      }
                }
            >
                English teacher at MDS.
            </h3>
            <div
                className="logo__photo__container"
            >
                <img
                    className="logo__photo"
                    onClick={[Function]}
                    src="https://firebasestorage.googleapis.com/v0/b/resume-40a8a.appspot.com/o/fotos%2FMDS.jpg?alt=media&token=04363ae9-87aa-4c09-be5d-4dd2f29a6e0e"
                    style={
                        Object {
                            "cursor": "pointer",
                          }
                    }
                />
            </div>
        </div>
        <div
            className="fecha_container"
        >
            <h3
                className="fecha__inicio"
            >
                2009
                 -
            </h3>
            <h3
                className="fecha__fin"
            >
                -&gt;  
                2012
            </h3>
        </div>
        <div
            className="description__container"
        >
            <span
                className="description__text"
            >
                English teacher in different Spanish companies like Gonvarri, Dematic or Gestamp.
                <br />
            </span>
        </div>
    </div>
</div>,
  <div
    className="work__java"
>
    <div
        className="leaf__container"
    >
        <div
            className="timeBox__headline__container"
        >
            <h3
                className="timeBox__name"
                onClick={[Function]}
                style={
                    Object {
                        "cursor": "auto",
                      }
                }
            >
                Java Developer at Awimbawe LLC
            </h3>
            <div
                className="logo__photo__container"
            >
                <img
                    className="logo__photo"
                    onClick={[Function]}
                    src="https://firebasestorage.googleapis.com/v0/b/resume-40a8a.appspot.com/o/fotos%2Fjava.png?alt=media&token=d0c05b0b-656e-4159-8cc7-73c8c7ba0c4f"
                    style={
                        Object {
                            "cursor": "auto",
                          }
                    }
                />
            </div>
        </div>
        <div
            className="fecha_container"
        >
            <h3
                className="fecha__inicio"
            >
                2010
                 -
            </h3>
            <h3
                className="fecha__fin"
            >
                -&gt;  
                2010
            </h3>
        </div>
        <div
            className="description__container"
        >
            <span
                className="description__text"
            >
                collaboration in a project programming in Java with server Jboss.
                <br />
            </span>
        </div>
    </div>
</div>,
  <div
    className="work__programmingC"
>
    <div
        className="leaf__container"
    >
        <div
            className="timeBox__headline__container"
        >
            <h3
                className="timeBox__name"
                onClick={[Function]}
                style={
                    Object {
                        "cursor": "pointer",
                      }
                }
            >
                Software developer C++ at CITEF
            </h3>
            <div
                className="logo__photo__container"
            >
                <img
                    className="logo__photo"
                    onClick={[Function]}
                    src="https://firebasestorage.googleapis.com/v0/b/resume-40a8a.appspot.com/o/fotos%2Fcitef.png?alt=media&token=aa241e1e-52dd-4041-a422-82796a6dd7b5"
                    style={
                        Object {
                            "cursor": "pointer",
                          }
                    }
                />
            </div>
        </div>
        <div
            className="fecha_container"
        >
            <h3
                className="fecha__inicio"
            >
                2006
                 -
            </h3>
            <h3
                className="fecha__fin"
            >
                -&gt;  
                2008
            </h3>
        </div>
        <div
            className="description__container"
        >
            <span
                className="description__text"
            >
                part of the interface team building a train simulator, for the learning Madrid subway drivers.
                <br />
            </span>
        </div>
    </div>
</div>,
  <div
    className="work__android"
>
    <div
        className="leaf__container"
    >
        <div
            className="timeBox__headline__container"
        >
            <h3
                className="timeBox__name"
                onClick={[Function]}
                style={
                    Object {
                        "cursor": "pointer",
                      }
                }
            >
                Founder of Bilbao shopping on foot
            </h3>
            <div
                className="logo__photo__container"
            >
                <img
                    className="logo__photo"
                    onClick={[Function]}
                    src="https://firebasestorage.googleapis.com/v0/b/resume-40a8a.appspot.com/o/fotos%2FBSOF.png?alt=media&token=722dda9c-b232-4c8c-8ed6-ad5d640cfa14"
                    style={
                        Object {
                            "cursor": "pointer",
                          }
                    }
                />
            </div>
        </div>
        <div
            className="fecha_container"
        >
            <h3
                className="fecha__inicio"
            >
                2012
                 -
            </h3>
            <h3
                className="fecha__fin"
            >
                -&gt;  
                2015
            </h3>
        </div>
        <div
            className="description__container"
        >
            <span
                className="description__text"
            >
                Start-up based on a marketing campaign alongside a highlighted position in the local business and shop searching mobile app.
                <br />
            </span>
        </div>
    </div>
</div>,
  <div
    className="work__rope__acces__technician"
>
    <div
        className="leaf__container"
    >
        <div
            className="timeBox__headline__container"
        >
            <h3
                className="timeBox__name"
                onClick={[Function]}
                style={
                    Object {
                        "cursor": "pointer",
                      }
                }
            >
                Asaken Rope Access
            </h3>
            <div
                className="logo__photo__container"
            >
                <img
                    className="logo__photo"
                    onClick={[Function]}
                    src="https://firebasestorage.googleapis.com/v0/b/resume-40a8a.appspot.com/o/fotos%2FasakenCroped.jpg?alt=media&token=d4ef928d-be3b-44a6-b015-acd25967c9a1"
                    style={
                        Object {
                            "cursor": "pointer",
                          }
                    }
                />
            </div>
        </div>
        <div
            className="fecha_container"
        >
            <h3
                className="fecha__inicio"
            >
                2014
                 -
            </h3>
            <h3
                className="fecha__fin"
            >
                -&gt;  
                2016
            </h3>
        </div>
        <div
            className="description__container"
        >
            <span
                className="description__text"
            >
                very technical and interesting job in the field.
                <br />
            </span>
            <span
                className="description__text"
            >
                Oil rigs, wind farms, industrial factories.
                <br />
            </span>
            <span
                className="description__text"
            >
                During my last year,I got to be team leader in several projects.
                <br />
            </span>
        </div>
    </div>
</div>,
  <div
    className="work__web__app"
>
    <div
        className="leaf__container"
    >
        <div
            className="timeBox__headline__container"
        >
            <h3
                className="timeBox__name"
                onClick={[Function]}
                style={
                    Object {
                        "cursor": "pointer",
                      }
                }
            >
                Web App Developer freelance
            </h3>
            <div
                className="logo__photo__container"
            >
                <img
                    className="logo__photo"
                    onClick={[Function]}
                    src="https://firebasestorage.googleapis.com/v0/b/resume-40a8a.appspot.com/o/fotos%2Fmico_disegno_textil.jpg?alt=media&token=628ae993-7cb4-4382-84a4-635344bcc0d3"
                    style={
                        Object {
                            "cursor": "pointer",
                          }
                    }
                />
            </div>
        </div>
        <div
            className="fecha_container"
        >
            <h3
                className="fecha__inicio"
            >
                2017
                 -
            </h3>
            <h3
                className="fecha__fin"
            >
                -&gt;  
                still on it
            </h3>
        </div>
        <div
            className="description__container"
        >
            <span
                className="description__text"
            >
                Iâve been more than a year building web apps from scratch, my latest and biggest project is an e-comerce web app I'm building on my own.
                <br />
            </span>
            <span
                className="description__text"
            >
                Will be soon released :)
                <br />
            </span>
        </div>
    </div>
</div>,
  <div
    className="work__web__app"
>
    <div
        className="leaf__container"
    >
        <div
            className="timeBox__headline__container"
        >
            <h3
                className="timeBox__name"
                onClick={[Function]}
                style={
                    Object {
                        "cursor": "pointer",
                      }
                }
            >
                Web App Developer freelance
            </h3>
            <div
                className="logo__photo__container"
            >
                <img
                    className="logo__photo"
                    onClick={[Function]}
                    src="https://firebasestorage.googleapis.com/v0/b/resume-40a8a.appspot.com/o/fotos%2Fmico_disegno_textil.jpg?alt=media&token=628ae993-7cb4-4382-84a4-635344bcc0d3"
                    style={
                        Object {
                            "cursor": "pointer",
                          }
                    }
                />
            </div>
        </div>
        <div
            className="fecha_container"
        >
            <h3
                className="fecha__inicio"
            >
                2017
                 -
            </h3>
            <h3
                className="fecha__fin"
            >
                -&gt;  
                still on it
            </h3>
        </div>
        <div
            className="description__container"
        >
            <span
                className="description__text"
            >
                Iâve been more than a year building web apps from scratch, my latest and biggest project is an e-comerce web app I'm building on my own.
                <br />
            </span>
            <span
                className="description__text"
            >
                Will be soon released :)
                <br />
            </span>
        </div>
    </div>
</div>,
]
`;
